ketcher-autotests/tests/Structure-Creating-&-Editing/SMARTS-attributes/Query-specific-attributes/query-specific-attributes.spec.ts

```diff
@@ -175,4 +175,68 @@ test.describe('Checking query specific attributes in SMARTS format', () => {
       '[#6](-[#6])(-[#6;@@])-[#6]',
     );
   });
+
+  test('Setting implicit H count, aromacity and connectivity', async ({
+    page,
+  }) => {
+    /**
+     * Test case: https://github.com/epam/Indigo/issues/1321
+     * Description: saving SMARTS with implicit H count and any other attribute should not cause any error
+     */
+    const expectedSmarts = '[#6](-[#6])(-[c;h5;X2])-[#6]';
+    await setImplicitHCount(page, '5');
+    await setAromaticity(page, 'aromatic');
+    await setConnectivity(page, '2');
+    await pressButton(page, 'Apply');
+    await takeEditorScreenshot(page);
+    await checkSmartsValue(page, defaultFileFormat, expectedSmarts);
+    await takeEditorScreenshot(page);
+  });
+
+  test.describe('Checking converting attributes to custom query', () => {
+    test.beforeEach(async ({ page }) => {
+      const numberOfAtom = 0;
+      await waitForPageInit(page);
+      await drawStructure(page);
+      await page.keyboard.press('Escape');
+      await doubleClickOnAtom(page, 'C', numberOfAtom);
+      await waitForAtomPropsModal(page);
```

Comment:
Maybe we could exract a common function in this file or other util files for these llines to reduce replication since it is used more than once